### PR TITLE
Updating the roles to Zabbix 5.4

### PR DESCRIPTION
**Pull request by dj-wasabi**
##### SUMMARY

Updating the roles to Zabbix 5.4

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by dj-wasabi** [comment]
@D3DeFi / @sky-joker 

You want to review this one first before merging, or should I just merge it? 😃

**Review by D3DeFi — APPROVED** [review]
Looks good bro, maybe just the one typo, which may or may not be a bug needs to be checked :rocket:

File: roles/zabbix_proxy/tasks/Debian.yml [hunk 1]
```diff
@@ -23,14 +23,25 @@
...
   tags:
     - zabbix-proxy
     - init
     - config
 
+- name: "Debian | Set some facts for Zabbix >= 3.0 && < 5.4"
+  set_fact:
+    datafiles_path: /usr/share/doc/zabbix-sql-scripts/{{ zabbix_proxy_database_long }}
+  when:
+    - zabbix_version is version('5.4', '>=')
```
[the remark this answers is not in the document]
Comment (dj-wasabi) [reply roles/zabbix_proxy/tasks/Debian.yml]: Ah yes, fixed it! Thanks bro! 👍

[the remark this answers is not in the document]
Comment (D3DeFi) [reply roles/zabbix_proxy/tasks/Debian.yml]: Then feel free to merge :) I should really get to the release of 1.4.0 soon

**Pull request merged**